setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-install_reqs = ["Django>=1.4", "sixpack-client==1.0.0"]
+install_reqs = ["Django>=1.4", "sixpack-client==1.1.1"]
```

Comment:
I crossed check all the APIs used added and they are available in 1.4...is there anything else I should check?